backend/src/index.ts

```diff
@@ -384,6 +386,19 @@ handlers[API_ANIMATION_VIDEO] = async (input) => {
   return {response: new Response(result, responseOptions(CONTENT_TYPE_VIDEO_MP4))};
 };
 
+handlers[API_PROFILE_AVATAR] = async (input) => {
+  const result = await db.get(`profile/avatar:${expectUuidParam(input, "id")}`, "arrayBuffer");
+  // TODO how to handle content type?
```

Comment:
As far as the content type goes, let's just pretend it's a JPEG always for now. It'll most likely work for every image type as the browser is pretty tolerant of that, and in the future I only want it to be jpegs